src/serializer.js

```diff
@@ -440,12 +466,21 @@ var Serializer = (function () {
       case 'Variable':
         return expr.toNT()
       case 'Literal':
+        var val = expr.value.toString() // should be a string already
         if (expr.datatype && this.flags.indexOf('x') < 0) { // Supress native numbers
           switch (expr.datatype.uri) {
+
             case 'http://www.w3.org/2001/XMLSchema#integer':
-              return expr.value.toString()
+              return val
+
+            case 'http://www.w3.org/2001/XMLSchema#decimal': // In urtle must have dot
```

Comment:
Thanks. I should have been more specific by mentioning that the line *below* this one could use a test case (the appending of `'.0'`)